webapp/src/main/java/org/opentestsystem/rdw/admin/model/Group.java

```diff
@@ -3,11 +3,20 @@
 import org.opentestsystem.rdw.common.model.Subject;
 
 public class Group {
+    private int id;
```

Comment:
If this is an entity ID, generally we have been using type long for good measure